tstest/natlab/vnet/conf.go

```diff
@@ -285,17 +296,18 @@ func (s *Server) initFromConfig(c *Config) error {
 			LinkType: layers.LinkTypeIPv4,
 		}))
 	}
-	for i, conf := range c.nodes {
+	for _, conf := range c.nodes {
 		if conf.err != nil {
 			return conf.err
 		}
 		n := &node{
+			num:           conf.num,
 			mac:           conf.mac,
 			net:           netOfConf[conf.Network()],
 			verboseSyslog: conf.VerboseSyslog(),
 		}
 		n.interfaceID = must.Get(s.pcapWriter.AddInterface(pcapgo.NgInterface{
-			Name:     fmt.Sprintf("node%d", i+1),
+			Name:     fmt.Sprintf("node%d", n.num),
```

Comment:
you could do n.String() here now

Reply:
It's `node` that has the String method, not the conf `Node`

Reply:
ah